docs/_includes/breadcrumb.html

```diff
@@ -0,0 +1,49 @@
+<div id="breadcrumb">
+
+    <h2><span class="secondary-text">@ebay/skin/</span>breadcrumb</h2>
+
+    <h3>Breadcrumb - Last item as the parent</h3>
+    <p>The last item in the breadcrumb list is the parent of the current page. Use this pattern if the current page category is not specific</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <ul class="breadcrumb">
```

Comment:
I feel like the root tag should be a `<nav>`. Breadcrumbs are always navigational landmarks.

Reply:
Oh, and when/if we do change this to a nav element, it will need to be labelled:

```html
<nav class="breadcrumb" aria-labelledby="breadcrumb-label">
  <h2 class="breadcrumb__heading" id="breadcrumb-label">You are here</h2>
  <ul>
    ..
  </ul>
</nav>
```

Please see the a11y pattern: https://ebay.gitbooks.io/mindpatterns/content/navigation/breadcrumbs.html

Note that the pattern needs to be updated (by me) to accommodate background SVG for the icons.